### PR TITLE
Django 2.1

### PR DESCRIPTION
**Pull request by glogiotatidis**
- Upgrade to Django 2.1
- Removed django-filters and pages listing published snippets on `/` b/c there's no-one watching those now. When rebuild we will use another filtering method with JEXL
- Removed admin advanced filters b/c they are not used.

### CODE REVIEW

File: snippets/base/urls.py [hunk 1]
```diff
@@ -7,25 +7,18 @@
 
 
 urlpatterns = [
-    url(r'^$', views.SnippetIndexView.as_view(), name='base.index'),
-    url(r'^(?P<startpage_version>[^/]+)/(?P<name>[^/]+)/(?P<version>[^/]+)/'
```
Comment (glogiotatidis) [inline snippets/base/urls.py]: loving how you can cast url variables now. Here making `startpage_version` an int.

Comment (jpetto) [reply snippets/base/urls.py]: Oooh!

Comment (pmac) [reply snippets/base/urls.py]: Ooooooooo! So this is something the new `django.urls.path` can do? I need to read up on this. Looks nice! A bit more Flasky.

Comment (pmac) [reply snippets/base/urls.py]: Oh yes. It's very nice. https://docs.djangoproject.com/en/2.1/topics/http/urls/#path-converters

Need to get a project up to Django 2.0+ :)

**Comment by glogiotatidis** [comment]
Updates

**Comment by pmac** [comment]
Looks like your build on Travis got bit by https://github.com/redis/hiredis-py/issues/38. They recently uploaded a bunch of wheels for `hiredis` which breaks our hashes. Just need to `hashin` again to get the new hashes.

**Comment by glogiotatidis** [comment]
Thanks pmac!

**Pull request merged**